jams/schemata/namespaces/pitch/note_midi.json

```diff
@@ -0,0 +1,8 @@
+{"note_midi":
+    {
+        "value": { "type": "number" },
```

Comment:
Not that I know of.  Is MIDI-(this JAMS format)-MIDI intended to be effectively lossless?  Note that standard MIDI pitch bend range is [-2, +2) semitones, so you'd actually need to do [0, 129) (or [-2, 129)?) to cover all possible pitch bends of MIDI notes.